janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/CQLStoreManager.java

```diff
@@ -185,6 +186,13 @@ public CQLStoreManager(final Configuration configuration) throws BackendExceptio
         fb.optimisticLocking(true);
         fb.multiQuery(false);
 
+        Boolean externalLocking = configuration.get(EXTERNAL_LOCKING);
```

Comment:
make final and change name to `hasExternalLocking`

Reply:
Done